### ISSUE TITLE
Error -32602: Invalid params.: Invalid parameter "/output/2": value must be one of "proxyid", "name" ...

### ISSUE THREAD

**Issue by tjyang** [unioslo/zabbix-cli#169]
- not able to connect to  API  server on zabbix 7.0 alpha 7
```
[ERROR]:('Error -32602: Invalid params.: Invalid parameter "/output/2": value must be one of "proxyid", "name", "operating_mode", "description", "allowed_addresses", "address", "port", "tls_connect", "tls_accept", "tls_issuer", "tls_subject", "custom_timeouts", "timeout_zabbix_agent", "timeout_simple_check", "timeout_snmp_agent", "timeout_external_check", "timeout_db_monitor", "timeout_http_agent", "timeout_ssh_agent", "timeout_telnet_agent", "timeout_script", "lastaccess", "version", "compatibility". while sending {\'jsonrpc\': \'2.0\', \'method\': \'proxy.get\', \'params\': {\'output\': [\'proxyid\', \'host\']}, \'id\': 4, \'auth\': \'f4456ef6e1292fe3f20fb9f972d2096f\'}', -32602)

```
- [7.0 API Doc](https://www.zabbix.com/documentation/devel/en/manual/api)

**Comment by pederhan** [comment]
It seems like a bunch of proxy-related properties ([Proxy.name](https://www.zabbix.com/documentation/devel/en/manual/api/reference/proxy/object#:~:text=for%20update%20operations-,name,-string), [Host.proxyid](https://www.zabbix.com/documentation/7.0/en/manual/api/reference/host/object#host:~:text=host%20property%20value.-,proxyid,-ID), and more?) have been renamed in 7.0. 

I have a branch with a temporary fix here: https://github.com/pederhan/zabbix-cli/tree/fix-login-7-0-bandaid

I am working on a more robust solution that solves this a bit more elegantly, but I need to actually understand the full extent of what was changed in 7.0 first.

**Comment by tjyang** [comment]
@pederhan 

Big thanks for 7.0 API fix, It almost worked.

- install login as root user on t01 rockylinux 8
```
pip install git+https://github.com/pederhan/zabbix-cli.git@fix-login-7-0-bandaid
```
- Using regualrs user "me" with proper auth and zabbix-cli.conf config files.

```
[me@t01 ~]$ ls -l .zabbix-cli_auth .zabbix-cli/zabbix-cli.conf
-r-------- 1 tjyang domain users  14 Nov 25 10:16 .zabbix-cli_auth
-rw-r--r-- 1 tjyang domain users 595 Nov 25 10:18 .zabbix-cli/zabbix-cli.conf
[me@t01 ~]$

```
- New error message on 'host'

```
[me@t01 ~]$  zabbix-cli
<snipped>
/usr/lib/python3.6/site-packages/urllib3/connectionpool.py:1052: InsecureRequestWarning: Unverified HTTPS request is being made to host 'ilclnagios03.mot.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,

[ERROR]:'host'

[me@t01 ~]$ 
```

**Comment by pederhan** [comment]
Oh, that's weird. It worked for me on my empty local Zabbix 7.0 instance, but I encounter the same error now after adding a proxy to it.

**Comment by tjyang** [comment]
@pederhan , you are correct. My test zabbix-server have 5 proxy severs configured.

**Comment by pederhan** [comment]
Sorry for the inconvenience, but since the bandaid fix didn't pan out, I'm just going to keep these new changes in the main branch I'm working on: https://github.com/pederhan/zabbix-cli/tree/fix-login-7-0

Try it out to see if it solves the issue for you.

**Comment by tjyang** [comment]
I am grateful of your effort and fix. I will keep checking the latest comment at https://github.com/pederhan/zabbix-cli/commits/fix-login-7-0. To see if it fix the zabbix 7 alpha 7 with proxy servers configured.

**Comment by tjyang** [comment]
@pederhan ,Thanks for the fixes in [fix-login-7-0](https://github.com/pederhan/zabbix-cli/commits/fix-login-7-0).
I can now connect to 7.0a7 zabbix server with proxy servers configured. 

- Successful connection banner after clear command

```
#############################################################
Welcome to the Zabbix command-line interface (v2.3.1)
Connected to server https://test03/zabbix (v7.0.0)
#############################################################
Type help or \? to list commands.

[zabbix-cli Admin@test03]$


```

- shell command not working

```
[zabbix-cli Admin@test03.test.com]$ shell uname

[Error]: Problems running uname


[zabbix-cli Admin@test03$

```

**Comment by pederhan** [comment]
Fixed in #173
